### PR TITLE
feat: add new resource for next.js

### PR DESCRIPTION
**Pull request by rwlc**
## What does this PR do?
Add resource(s) 
### Why is this valuable (or not)?
The next.js team has a _really_ good tutorial on their official website — and I believe it’s one of the best sources to learn about it.

### How do we know it's really free?

It’s on a public website, and open source.

### For book lists, is it a book? For course lists, is it a course? etc.

It’s kind-of a book. It’s not really compiled into a PDF, but rather a site.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1244,6 +1244,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Mastering Next.js](https://masteringnextjs.com)
 * [Next.js Crash Course 2021](https://www.youtube.com/watch?v=mTz0GXj8NN0) - Brad Traversy (Traversy Media)
 * [The Next.js Handbook](https://flaviocopes.com/page/nextjs-handbook/) - Flavio Copes (PDF, EPUB, Kindle) *(email address requested)*
+* [The official Next.js guide](https://nextjs.org/learn) - The official Next.js team
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1247]: Don't invent titles! Use the title given by the resource. Same with creators - this one may be a corporate author.

Comment (rwlc) [reply books/free-programming-books-langs.md:1247]: Oho, my bad. There isn't really a title to the book, per se -- but what the creators usually call it is the "Offical Next.js guide" while referencing to it.

Plus, should I just remove the authors, then? It isn't really public knowledge on who's really writing them -- it's just marketed by the entire vercel team.

Comment (eshellman) [reply books/free-programming-books-langs.md:1247]: we need to be able to see if the resource is a duplicate by the title and creators. I think the author here is the company name and look at the page for title - e.g. the html title

Comment (davorpa) [reply books/free-programming-books-langs.md:1247]: As requested

```suggestion
* [Learn Next.js](https://nextjs.org/learn) - Vercel Inc.
```

Comment (rwlc) [reply books/free-programming-books-langs.md]: Done!

**Review by davorpa — APPROVED** [review]

**Pull request merged**